src/main/java/uk/gov/pay/ledger/event/model/SalientEventType.java

```diff
@@ -22,6 +22,7 @@
     USER_APPROVED_FOR_CAPTURE_AWAITING_SERVICE_APPROVAL,
     SERVICE_APPROVED_FOR_CAPTURE,
     STATUS_CORRECTED_TO_CAPTURED_TO_MATCH_GATEWAY_STATUS,
+    CAPTURE_NOTIFICATION_FOR_EXPUNGED_CHARGE,
```

Comment:
👋significant events so far have used semantic, past tense naming to describe exactly what happened.

As this maps to an external `SUCCESS` state I'm guessing this means the payment was captured by the gateway? Making this clear will bring this more in line with other event names here. 

Potentially it could be something like: 
`CAPTURE_CONFIRMED_BY_GATEWAT_NOTIFICATION` 

Happy to chat about this if it doesn't line up with the rest of the naming for this feature.

Reply:
Sure - how about EXPUNGED_CHARGE_STATUS_CORRECTED_TO_CAPTURE_TO_MATCH_GATEWAY_STATUS